workloads/files/workload-mastervertical-script-cm.yml

```diff
@@ -80,7 +80,10 @@ data:
       result_dir=${benchmark_results_dir}
     fi
     start_time=$(date +%s)
-    VIPERCONFIG=/tmp/mastervertical.yaml openshift-tests run-test "[Feature:Performance][Serial][Slow] Load cluster concurrently with templates [Suite:openshift]" | tee "${result_dir}/clusterloader.txt"
+    export es={{ snafu_es_host }}
```

Comment:
@chaitanyaenr is this the right way to use the ansible variable ? or am i missing something else as well?